apps/web/app/components/ui/card.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from "react"
+
+import { cn } from "@/web/lib/utils";
+
+const Card = React.forwardRef<
```

Comment:
Since we're using React v19, [`forwardRef` is not needed anymore](https://react.dev/blog/2024/12/05/react-19#ref-as-a-prop).